Packages/com.unity.inputsystem/InputSystem/Utilities/NameAndParameters.cs

```diff
@@ -27,6 +27,24 @@ public override string ToString()
             return $"{name}({parameterString})";
         }
 
+        internal static string SerializeMultiple(IEnumerable<NameAndParameters> list)
+        {
+            if(list == null)
+                return string.Empty;
+
+            return string.Join(NamedValue.Separator, list.Select(x => x.ToString()).ToArray());
+        }
+
+        internal static NameAndParameters Create(string name, IList<NamedValue> parameters)
+        {
+            var result = new NameAndParameters
```

Comment:
Nitpick: Why create a result variable here instead of just 

return new NameAndParameters { .... };